ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/DownloadFile.kt

```diff
@@ -0,0 +1,392 @@
+/*
+ This file is part of Subsonic.
+
+ Subsonic is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Subsonic is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Subsonic.  If not, see <http://www.gnu.org/licenses/>.
+
+ Copyright 2009 (C) Sindre Mehus
```

Comment:
@ogarcia once warned me to replace this with a new copyright when the file is converted to kotlin... I'm still not sure what is the best practice here, I've since seen in other open source projects that they just write something like "Copyright (C) 2009-2021 Ultrasonic Developers", this way the actual contributors can be found in Git.
What do you think?

Reply:
Yes.
And we must use a short form:
```kotlin
/*
 * DownloadFile.kt
 * Copyright (C) 2009-2021 Ultrasonic developers
 *
 * Distributed under terms of the GNU GPLv3 license.
 */
```

But note that if is a new code (and not a refactor) we must set copyright line to this year.
/*
 * NameOfFile.kt
 * Copyright (C) 2021 Ultrasonic developers
 *
 * Distributed under terms of the GNU GPLv3 license.
 */

Reply:
Ok, done